src/components/button.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function button(props) {
+  return (
+    <div>
+      <h1>
+        {props.name}
+      </h1>
+    </div>
+  );
+}
+
+export default button;
```

Comment:
- The button name prop should be a String.

`Button.propTypes = {
  name: PropTypes.string
};`